js/renovation/ui/box/__tests__/box.test.tsx

```diff
@@ -20,4 +26,68 @@ describe('Box > Attrs', () => {
       expect(classList.contains('dx-box-flex')).toEqual(true);
     });
   });
+
+  each([1, 'text', true, undefined, null]).describe('cssStyles: %o', (style) => {
+    it('Should pass styles to container from cssStyles', () => {
+      const box = mount<Box>(viewFunction({ cssStyles: style } as any));
+      const rootContainer = box.find(Widget);
+
+      expect(rootContainer.props().style).toEqual(style);
+    });
+  });
+
+  describe('cssStyles', () => {
+    it('display style', () => {
+      const props = {} as BoxProps;
+      const box = new Box(props);
+      expect(box.cssStyles.display).toEqual('flex');
+    });
+
+    each(['row', 'column', undefined]).describe('direction: %o', (direction) => {
+      it('flexDirection style', () => {
+        const props = { direction } as BoxProps;
+        const box = new Box(props);
+        let expectedDirection;
+        if (direction === 'row') {
+          expectedDirection = 'row';
+        } else if (direction === 'col') {
+          expectedDirection = 'column';
+        }
+
+        expect(box.cssStyles.flexDirection).toEqual(expectedDirection);
+      });
+    });
+
+    each(['center', 'end', 'space-around', 'space-between', 'start', undefined]).describe('align: %o', (align) => {
+      it('justifyContent style', () => {
+        const props = { align } as BoxProps;
+        const box = new Box(props);
+
+        let justifyContent = align;
+        if (align === 'start') {
+          justifyContent = 'flex-start';
+        } else if (align === 'end') {
+          justifyContent = 'flex-end';
+        }
+
+        expect(box.cssStyles.justifyContent).toEqual(justifyContent);
+      });
+    });
+
+    each(['center', 'end', 'start', 'stretch', undefined]).describe('crossAlign: %o', (crossAlign) => {
+      it('alignItems style', () => {
+        const props = { crossAlign } as BoxProps;
+        const box = new Box(props);
+
+        let alignItems = crossAlign;
+        if (crossAlign === 'start') {
+          alignItems = 'flex-start';
+        } else if (crossAlign === 'end') {
+          alignItems = 'flex-end';
+        }
+
+        expect(box.cssStyles.alignItems).toEqual(alignItems);
+      });
+    });
```

Comment:
imho this is better than 'each':

      it('crossAlign', () => {
        const props = {} as BoxProps;
        props.crossAlign = 'center';
        expect(new Box(props).cssStyles.alignItems).toEqual('center');

        props.crossAlign = 'end';
        expect(new Box(props).cssStyles.alignItems).toEqual('flex-end');

        props.crossAlign = 'start';
        expect(new Box(props).cssStyles.alignItems).toEqual('flex-start');

        props.crossAlign = 'stretch';
        expect(new Box(props).cssStyles.alignItems).toEqual('stretch');

        props.crossAlign = undefined;
        expect(new Box(props).cssStyles.alignItems).toEqual(undefined);

        props.crossAlign = null;
        expect(new Box(props).cssStyles.alignItems).toEqual(null);
      });

Reply:
Yep, fixed